registry/infrastructure/models/models.py

```diff
@@ -138,3 +138,81 @@ class OrganizationAddressHistory(Base):
     country = Column("country", VARCHAR(64), nullable=False)
     created_on = Column("created_on", TIMESTAMP(timezone=False))
     updated_on = Column("updated_on", TIMESTAMP(timezone=False))
+
+
+# DB Model Class for Service
+
+class Service(Base):
+    __tablename__ = "service"
+    row_id = Column("row_id", Integer, primary_key=True, autoincrement=True)
+    uuid = Column("uuid", VARCHAR(128))
+    display_name = Column("display_name", VARCHAR(128), nullable=False)
+    service_id = Column("service_id", VARCHAR(128), nullable=False)
+    metadata_ipfs_hash = Column("metadata_ipfs_hash", VARCHAR(255))
+    proto = Column("proto", JSON, nullable=False)
+    short_description = Column("short_description", VARCHAR(1024), nullable=False)
+    description = Column("description", VARCHAR(1024), nullable=False)
+    git_url = Column("git_url", VARCHAR(512), nullable=False)
+    assets_hash = Column("assets_hash", JSON, nullable=False)
+    assets_url = Column("assets_url", JSON, nullable=False)
```

Comment:
merge` asset_hash` and `asset_url` into on json
```
{
    "asset_type": {
        "url": "",
        "hash":""
    }
}
```

Reply:
resolved. Agreed.